umm-spec-lib/src/cmr/umm_spec/test/iso_smap_expected_conversion.clj

```diff
@@ -232,17 +247,50 @@
          (map cmn/map->DataCenterType)
          distinct)))
 
+(defn- not-provided-begin-date
+  "Returns the default java epoch zero used for not provided date values.
+   This function returns the def record and not just a map so that
+   it can be compared to the actual round trip translation."
+  []
+  (for [x (range 0 1)]
```

Comment:
In order to compare the values with the actual I needed the entire defrecord to be in a vector or sequence. Using vec or seq turned every element in the defrecord into a vector and so I couldn't compare the values.  I don't know of any other way to get the result I needed. I tried using vec and seq together, by themselves, in the function, after the function, but I couldn't get the structure I needed.